build/ci/wait_for_k8s_ready.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -xe
+set +o pipefail
+
+is_kubernetes_cluster_ready (){
+  pods=`kubectl get pods -A | awk '{print$3}' | grep -iv ready`
+  all_the_containers_are_runninig=true
+  for pod in $pods; do
+    running_containers=`echo $pod | awk -F / '{print$1}'`
+    wanted_containers=`echo $pod | awk -F / '{print$2}'`
+    if [ $running_containers != $wanted_containers ]; then
+      all_the_containers_are_runninig=false
```

Comment:
no need for `the`
`runninig` -> `running` (please use an IDE with a spellchecker)

Reply:
done